src/openrct2/ride/RideData.cpp

```diff
@@ -783,117 +783,6 @@ const uint8_t rideBonusValue[RIDE_TYPE_COUNT] = {
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0a Steeplechase/Motorbike/Soap Box Derby
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0b Car Ride
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0c Launched Freefall
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0d Bobsleigh Coaster
-    { RIDE_COMPONENT_TYPE_CABIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0e Observation Tower
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 0f Looping Roller Coaster
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 10 Dinghy Slide
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 11 Mine Train Coaster
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 12 Chairlift
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 13 Corkscrew Roller Coaster
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 14 Maze
-    { RIDE_COMPONENT_TYPE_TRAIN,        RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 15 Spiral Slide
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 16 Go Karts
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 17 Log Flume
-    { RIDE_COMPONENT_TYPE_BOAT,         RIDE_COMPONENT_TYPE_TRACK,              RIDE_COMPONENT_TYPE_STATION }, // 18 River Rapids
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 19 Dodgems
-    { RIDE_COMPONENT_TYPE_SHIP,         RIDE_COMPONENT_TYPE_STRUCTURE,          RIDE_COMPONENT_TYPE_STATION }, // 1a Pirate Ship
-    { RIDE_COMPONENT_TYPE_SHIP,         RIDE_COMPONENT_TYPE_STRUCTURE,          RIDE_COMPONENT_TYPE_STATION }, // 1b Swinging Inverter Ship
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1c Food Stall
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1d (none)
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1e Drink Stall
-    { RIDE_COMPONENT_TYPE_CAR,          RIDE_COMPONENT_TYPE_BUILDING,           RIDE_COMPONENT_TYPE_STATION }, // 1f (none)
```

Comment:
What were these nones? Did we never know or did we just removed/merged them?

Reply:
1f has always been an unknown stall that hasn't been used. I think same goes for the others

Reply:
Exactly. All the non-existing ride types use `DummyRTD`.